keps/sig-storage/625-csi-migration/README.md

```diff
@@ -156,10 +163,166 @@ Running this set of tests is optional for a per-provider basis. We would
 recommend it for providing extra confidence but the framework for
 upgrade/downgrade is provider agnostic.
 
+## Production Readiness Review Questionnaire
+
+<!--
+
+Production readiness reviews are intended to ensure that features merging into
+Kubernetes are observable, scalable and supportable; can be safely operated in
+production environments, and can be disabled or rolled back in the event they
+cause increased failures in production. See more in the PRR KEP at
+https://git.k8s.io/enhancements/keps/sig-architecture/1194-prod-readiness.
+
+The production readiness review questionnaire must be completed and approved
+for the KEP to move to `implementable` status and be included in the release.
+
+In some cases, the questions below should also have answers in `kep.yaml`. This
+is to enable automation to verify the presence of the review, and to reduce review
+burden and latency.
+
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+
+-->
+
+### Feature Enablement and Rollback
+
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: CSIMigration, CSIMigration{cloud-provider}
+    - Components depending on the feature gate: kubelet, kube-controller-manager
+
+* **Does enabling the feature change any default behavior?**
+  Yes and No. If only CSIMigration feature flag is enabled, nothing will change on the cluster behavior. However, if CSIMigration && CSIMigration{cloud-provider} are both enabled, the behavior will change. The in-tree volume plugin that the cloud-provider use will be redirect to use the corresponding CSI driver. But from a user perspective, nothing will be noticed.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+  Yes. Once there is an error or something unexpected, users can disable the CSI migration feature flags to roll it back.
```

Comment:
nit: change change to:

"Yes - can be disabled by disabling feature flags"

That said - questions:
- any concerns about state stored in etcd? Anything that CSI drivers will report differently than in-tree drivers that we may have problems to reconcile after rollback? @msau42 fpr thoughts too

Reply:
I dont think we have anything special that will be stored in etcd after the feature is turned on. It is mainly a few annotations of PV, and for those we will remove them when the migration is turned from on to off.

Reply:
PV annotations have been tested already in rollback scenarios.

Can you link to the section about the steps to enable/disable the feature. There are some strict requirements in ordering and steps (such as draining the node first), so that we don't end up with stale VolumeAttachment objects.

In general, there is some risk about state related to migration in PV and VolumeAttachment objects, so we need to make sure rollback and disablement is tested thoroughly (and has already been done back in 1.17 timeframe). @Jiawei0227 can you make sure the subtle issues are documented in the rollback section?

Reply:
Sure, I will revisit these section